rtd-environment.yml

```diff
@@ -5,5 +5,3 @@ dependencies:
   - openjpeg
   - pip
   - python=3.9
-  - pip:
-      - sphinx-gallery
```

Comment:
Since rtd installs this via pip install sunpy, we don't need to actually do this here anymore at all.